ui/pages/home/home.component.js

```diff
@@ -856,6 +880,11 @@ export default class Home extends PureComponent {
           {
             ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
           }
+          <SmartTransactionsModal
+            onEnable={this.onEnableStx}
```

Comment:
Given this is a root component in the UI, should we be avoiding coupling it to more domain wherever possible?

I appreciate it's difficult to route to this modal without logic here, but could we at least encapsulate the handlers and actions within the `SmartTransactionModal` itself rather than here and the associated container mappings.